functions/slack/search.go

```diff
@@ -83,6 +92,20 @@ func verifyWebHook(form url.Values) error {
 	return nil
 }
 
+func verifyWebhookWithSigning(r *http.Request) error {
```

Comment:
Just `verifyWebhook`? Don't need to show both options for verifying it, just the "right" one (signing).